tsl/org.testeditor.tsl.dsl.tests/src/org/testeditor/tsl/dsl/parser/MarkupParserTest.xtend

```diff
@@ -0,0 +1,146 @@
+package org.testeditor.tsl.dsl.parser
+
+import org.eclipse.xtext.junit4.XtextRunner
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.testeditor.tsl.util.TslModelUtil
+import com.google.inject.Inject
+import org.testeditor.tsl.TextLine
+import org.testeditor.tsl.BoldText
+import org.testeditor.tsl.BoldCursiveText
+import org.testeditor.tsl.CursiveText
+
+@RunWith(XtextRunner)
+class MarkupParserTest extends AbstractParserTest {
+
+	@Inject extension TslModelUtil
+
+	@Test
+	def void missingPackage(){
+		// given
+		val tsl = '''
+			// package packageA
+		'''
+		
+		// expect nothing parsed
+		tsl.parse[
+			assertNull	
+		]
+	}
+	
+	@Test
+	def void packageAlone(){

```

Comment:
This test should be removed, it does the same as `SimpleTslParserTest.parseEmptyModel`